src/screens/Auth/RegisterStack/UserInfoScreen/styles.tsx

```diff
@@ -0,0 +1,98 @@
+import { StyleSheet } from 'react-native'
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+  },
+
+  header: {
+    flex: 1.5,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    color: '#000',
+    marginTop: '10%'
+  },
+
+  main: {
+    flex: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    color: '#000',
+  },
+
+  footer: {
+    flex: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    color: '#000',
+  },
+
+  subsection: {
+    flex: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'flex-start',
+    color: '#000',
+  },
+
+  catagories: {
+    flex: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    fontFamily: 'montserrat-semibold',
+    fontSize: 16,
+    lineHeight: 20,
+    color: '#000000'
+  },
+
+  valuesText: {
+    flex: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    fontFamily: 'montserrat-regular',
+    fontSize: 22,
+    lineHeight: 27,
+    color: '#E5186E',
+  },
+
+  bar: {
+    flex: .25,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',

```

Comment:
@vikram-kher forgot this in the last review, but these lines are also unnecessary